content/notes/config.md

```diff
@@ -23,6 +24,49 @@ links: # Links to show in footer
     link: https://github.com/jackyzha0
 ```
 
+### Code Block Titles
+
+Quartz relies on Hugo's code highlighter,
+[Chroma](https://github.com/alecthomas/chroma), which happens to be ["really,
+really fast"](https://gohugo.io/content-management/syntax-highlighting/). As
+fast as it is, it does leave much to be desired, such as code block titles. In
+Obsidian, there's a plugin which provides this feature, called [Obsidian Better
+Code Block](https://github.com/stargrey/obsidian-better-codeblock). The idea
+behind the syntax is similar to Hugo's, although unfortunately not identical
+(stay tuned for a [Quartz
+binary](https://github.com/jackyzha0/quartz/issues/81)). To add code titles
+with Quartz:
+
+1. Ensure that code block titles are enabled in Quartz's configuration:
+
+    ```yaml {title="data/config.yaml", linenos=false}
+    enableCodeBlockTitle: true
+    ```
+
+2. Add the `title` attribute to the desired code block. Note there are two ways of
+   defining code blocks in Hugo.
+
+   - [**Highlight Shortcode**](https://gohugo.io/content-management/syntax-highlighting/#example-highlight-shortcode): Example not included as Hugo will insist on attempting to render it.
+   - [**Highlight Code Fences**](https://gohugo.io/content-management/syntax-highlighting/#highlighting-in-code-fences): Example provided 
+     intentionally omits a single back quote.
+
+      ```markdown {linenos=false}
+      ```yaml {title="data/config.yaml"}
```

Comment:
this may work by just having an indented code block https://www.jetbrains.com/help/hub/Markdown-Syntax.html#indented-code-blocks

e.g


	  ```
	  abc
	  ```